SequenceSnapshotGeneratorHana is being called for Snowflake leading to migration failure

Hana sequence priority ends up being positive, which leads to adding an incompatible SequenceSnapshotGeneratorHana
```
Caused by: liquibase.exception.UnexpectedLiquibaseException: Don't know how to query for sequences on DBEAVER @ jdbc:snowflake://nl30284.eu-central-1.snowflakecomputing.com:443/ (Default Schema: TEST1)
	at liquibase.snapshot.jvm.SequenceSnapshotGenerator.getSelectSequenceStatement(SequenceSnapshotGenerator.java:272)
	at liquibase.ext.hana.snapshot.SequenceSnapshotGeneratorHana.getSelectSequenceStatement(SequenceSnapshotGeneratorHana.java:34)
	at liquibase.snapshot.jvm.SequenceSnapshotGenerator.addTo(SequenceSnapshotGenerator.java:49)
	at liquibase.snapshot.jvm.JdbcSnapshotGenerator.snapshot(JdbcSnapshotGenerator.java:78)
	at liquibase.snapshot.SnapshotGeneratorChain.snapshot(SnapshotGeneratorChain.java:60)
	at liquibase.snapshot.jvm.JdbcSnapshotGenerator.snapshot(JdbcSnapshotGenerator.java:68)
	at liquibase.snapshot.SnapshotGeneratorChain.snapshot(SnapshotGeneratorChain.java:60)
	at liquibase.snapshot.DatabaseSnapshot.include(DatabaseSnapshot.java:313)
	at liquibase.snapshot.DatabaseSnapshot.init(DatabaseSnapshot.java:106)
	at liquibase.snapshot.DatabaseSnapshot.<init>(DatabaseSnapshot.java:59)
	at liquibase.snapshot.JdbcDatabaseSnapshot.<init>(JdbcDatabaseSnapshot.java:37)
	at liquibase.snapshot.SnapshotGeneratorFactory.createSnapshot(SnapshotGeneratorFactory.java:214)
	at liquibase.snapshot.SnapshotGeneratorFactory.createSnapshot(SnapshotGeneratorFactory.java:187)
	at com.dbeaver.ee.scmp.impl.liquibase.LBCompareEngine.createDatabaseSnapshot(LBCompareEngine.java:709)
	... 9 more
```
I am going to provide the fix in the pull request

--- a/src/main/java/liquibase/ext/hana/snapshot/SequenceSnapshotGeneratorHana.java
+++ b/src/main/java/liquibase/ext/hana/snapshot/SequenceSnapshotGeneratorHana.java
@@ -13,11 +13,12 @@ public class SequenceSnapshotGeneratorHana extends SequenceSnapshotGenerator {
 
     @Override
     public int getPriority(Class<? extends DatabaseObject> objectType, Database database) {
-        int priority = super.getPriority(objectType, database);
         if (database instanceof HanaDatabase) {
-            priority += PRIORITY_DATABASE;
+            int priority = super.getPriority(objectType, database);
+            return priority + PRIORITY_DATABASE;
+        } else {
+            return PRIORITY_NONE;
         }
-        return priority;
     }
 
     @Override

